app/utils/utils.js

```diff
@@ -1,4 +1,5 @@
 const bcrypt = require("bcrypt");
+const { check, validationResult } = require("express-validator");
 
 exports.saltGenerator = function () {
```

Comment:
This is wrong. The saltRounds should not be random. It should be around 13-14. Please read about bcrypt.

Reply:
Read about bcrypt and yes changing the salt rounds to 13.